basics/processing-instructions/steel/program/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+use processing_instructions_api::prelude::*;
+use solana_program::msg;
+use steel::*;
+
+pub fn process_instruction(
+    _program_id: &Pubkey,
+    _accounts: &[AccountInfo],
+    instruction_data: &[u8],
+) -> ProgramResult {
+    msg!("Processing instruction");
+
+    let instruction = GoToTheParkData::try_from_bytes(instruction_data)?;
```

Comment:
use `parse_instruction()`, [create instruction in a different file](https://github.com/regolith-labs/steel?tab=readme-ov-file#file-structure) and import it here using mod.
ex. `go_to_park.rs`

Reply:
Done!